src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -64,6 +64,11 @@
     private static final String DISCARD_SAMPLE_MENU = "discardSamples";
     private static final String PDF_DOWNLOAD_MENU = "pdfDownload";
 
+    public static boolean isHruid(@NonNull String participantId) {
+        final String hruidCheck = "^P\\w{5}$";
+        return participantId.matches(hruidCheck);
+    }
+
```

Comment:
Not sure this should go in here. That class is for user as in study staff… I think we should make a ParticipantUtil

Reply:
Sounds good, will move it, thanks